android/src/app/organicmaps/maplayer/MapButtonsController.java

```diff
@@ -70,10 +71,12 @@ else if (mLayoutMode == LayoutMode.planning)
     mBottomButtonsFrame = mFrame.findViewById(R.id.map_buttons_bottom);
 
     final FloatingActionButton helpButton = mFrame.findViewById(R.id.help_button);
-    if (Config.isNY() && helpButton != null)
+    if (helpButton != null)
     {
-      helpButton.setImageResource(R.drawable.ic_christmas_tree);
-      helpButton.getDrawable().setTintList(null);
+      helpButton.setImageResource(R.drawable.logo);
+      // Keep this button colorful in normal theme.
```

Comment:
in my opinion, this doesn't look as good:
![signal-2023-02-19-210359](https://user-images.githubusercontent.com/26939824/219975295-646cbcff-cf08-477a-9480-09c76135d28e.png)

it might look better if the colour is changed to grey like the others?

Reply:
I bet that any user will tap an unknown icon at least once.

Reply:
> Yes, it is intentionally green to attract attention... 

> I bet that any user will tap an unknown button/icon at least once.

I also agree with @endim8 that this doesn't look good. Also the current question mark looks fine for me and it's much clearer that it's something related to "help/about".

For me the clean UI is one of the best things in OM. I understand that you try to get more users to read the about page, but I think many users really wouldn't like to have default UI of OM to be more "attention screaming" just to avoid some negative reviews on app stores. 

I have two alternative ideas:

1. Maybe some mixed approach would be valuable? Like having this button with green color after first install/update of application and render it in gray color after it's been clicked once? 

2. Or have possibility to disable this this button in options page? Some toggle that when it's ON then the 4 buttons change to 3 buttons (without the About Page), and the About Page appears just as another position when you click "hamburger menu" button. However, this might seem a bit like over-complicating setting page for not huge benefit. On the other hand there is already option like "hide zoom buttons" that affect what GUI element appear on app, so there is some precedence.